rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] Great job setting up associations for your classes. Let's complete the association here. A rental belongs to a book and also belongs to a person. A rental should add itself to both the @book and @person so as to acheive `belong-to` association. It should look something similar to this.

```
   def initialize(date, book, person)
      .
      .

      @book = book
      # *The rental is adding itself to the @book achieving the 
       'belongs-to' association
       book.rentals << self

       @person = person
       # *The rental is adding itself to the @person the 'belongs-to' 
       association
       person.rentals << self
    end
```

Reply:
@Qoosim  Done making changes